var/spack/repos/builtin/packages/llvm-openmp/package.py

```diff
@@ -47,7 +47,24 @@ class LlvmOpenmp(CMakePackage):
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
-    variant("multicompat", default=True, description="Support the GNU OpenMP runtime interface.")
+    variant(
+        "multicompat",
+        default=False,
+        description="Support gomp and the Intel openMP runtime library.",
+    )
+
+    # variant for building llvm-openmp as a stand alone library
+    variant(
+        "standalone",
+        default=False,
+        description="Build llvm openmpi ompt library as a \
+                         stand alone entity.",
```

Comment:
Format the file using `black`. Likely is must be like this (with a typo fix for `libomp`, AFAICS, and LLVM in uppercase, standalone should be one word, I think.):
```suggestion
        description="Build LLVM openmpi libomp library as standalone library.",
```

Reply:
I made this change.

Reply:
```suggestion
        description="Build llvm openmp ompt library as a stand alone entity.",
```

OpenMPI is an easy typo but a very different thing.

Reply:
Yes, I corrected that typo on my part.